samples/xamarin-forms/Cinephile/Cinephile/Views/UpcomingMoviesListView.xaml.cs

```diff
@@ -23,20 +20,24 @@ public UpcomingMoviesListView()
             {
                 ViewModel.SelectedItem = null;
 
-                this.OneWayBind(ViewModel, x => x.Movies, x => x.UpcomingMoviesList.ItemsSource)
-                    .DisposeWith(disposables);
-
-                this.Bind(ViewModel, x => x.SelectedItem, x => x.UpcomingMoviesList.SelectedItem)
-                    .DisposeWith(disposables);
+                this.OneWayBind(ViewModel, x => x.Movies, x => x.UpcomingMoviesList.ItemsSource).DisposeWith(disposables);
```

Comment:
Referring to the .xaml file, I have seen the same issue as AboutView and UpcomingMoviesCellView XAMLs. You have to specify the assembly like this:
```
xmlns:vms="clr-namespace:Cinephile.ViewModels;assembly=Cinephile.ViewModels"
```